src/test/resources/org/prebid/server/it/openrtb2/sharethrough/test-auction-sharethrough-response.json

```diff
@@ -0,0 +1,183 @@
+{
+  "id": "tid",
+  "seatbid": [
+    {
+      "bid": [
+        {
+          "adid": "arid",
+          "adm": "<img src=\"//b.sharethrough.com/butler?type=s2s-win&arid=arid\" />\n\t\t<div data-str-native-key=\"abc123\" data-stx-response-name=\"str_response_bid\"></div>\n\t\t<script>var str_response_bid = \"eyJhZHNlcnZlclJlcXVlc3RJZCI6ImFyaWQiLCJiaWRJZCI6ImJpZCIsImNyZWF0aXZlcyI6W3siY3BtIjoxMCwiY3JlYXRpdmUiOnsiY2FtcGFpZ25fa2V5IjoiY21wS2V5IiwiY3JlYXRpdmVfa2V5IjoiY3JlYUtleSIsImRlYWxfaWQiOiJkZWFsSWQifSwidmVyc2lvbiI6MH1dfQ==\"</script>\n\t\t\t<script src=\"//native.sharethrough.com/assets/sfp.js\"></script>\n",
+          "cid": "cmpKey",
+          "crid": "creaKey",
+          "dealid": "dealId",
+          "ext": {
+            "prebid": {
+              "cache": {
+                "bids": {
+                  "cacheId": "a03927f1-86e1-48fb-9efc-da9104a44e39",
+                  "url": "http://localhost:8090/cache?uuid=a03927f1-86e1-48fb-9efc-da9104a44e39"
```

Comment:
Please replace `http://localhost:8090/cache?uuid=` part of the url string with the `{{ cache.resource_url }}` placeholder.